File: ms-java/src/main/java/com/jakubeeee/playground/msjava/organization/internal/DepartmentRepository.java
package com.jakubeeee.playground.msjava.organization.internal;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DepartmentRepository extends JpaRepository<DepartmentEntity, Long> {

    DepartmentEntity getOneByName(String name);

}

File: ms-kotlin/src/main/kotlin/com/jakubeeee/playground/mskotlin/organization/internal/DepartmentRepository.kt
package com.jakubeeee.playground.mskotlin.organization.internal

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface DepartmentRepository : JpaRepository<DepartmentEntity, Long> {

    fun getOneByName(name: String): DepartmentEntity

}